docs/advanced/general/using-private-git.md

```diff
@@ -16,18 +17,35 @@ E-mail: github@vapor.codes
 
 Link to profile: https://github.com/vapor-cloud-api
 
-### Use SSH Key
+## Add your SSH Key to Vapor Cloud
+
+!!! info
```

Comment:
Is `!!!` a markdown feature?

Reply:
Yes, it adds a box in a color based on the text info, warning etc.